src/components/v2/V2Project/V2FundingCycleSection/modals/EditPayoutsModal.tsx

```diff
@@ -357,6 +364,12 @@ export const EditPayoutsModal = ({
       <DistributionSplitModal
         visible={addSplitModalVisible}
         onSplitsChanged={onSplitsChanged}
+        distributionLimit={
+          distributionLimit?.eq(MAX_DISTRIBUTION_LIMIT)
+            ? undefined
+            : formatWad(distributionLimit, { thousandsSeparator: '' })
+        }
+        overrideDistTypeWithBoth={true}
```

Comment:
no need for the `={true}`, it is implicit by being there

Reply:
nice catch ser 💯